### ISSUE TITLE
Contrast issue with colours affecting accessibility between the card element background and main website background-colour

### ISSUE THREAD

**Issue by hannahro15** [tom-ai/sophia-website#39]
An issue I have spotted is the background-colour of the card element which is dark grey and the background of the website which is black are very similar, affecting accessibility. Maybe more variation between the colours.
    It will probably show up as an issue you ran Google Lighthouse, or it you ran it on the WAVE accessibility website.

**Comment by hannahro15** [comment]
See issue 37 on the picture so you will see what I mean!

**Comment by tom-ai** [comment]
Good spot Hannah! I should have mentioned however, this website is not currently live. I switched to Wordpress last year. However I do want to keep this project as a template for future projects.

So feel free to keep to contribute to this still, including updating the colour of the card. And we'll look at updating the data so it's someone generic :)
